src/Slider.jsx

```diff
@@ -117,19 +117,7 @@ class Slider extends React.Component {
       this.setState({[state.handle]: value});
     }
 
-    if (props.range) {
-      // `this.state` will not be updated immediately after `this.setState`.
-      // So, create a similar object.
-      // const data = Object.assign({}, state, {[state.handle]: value});
-      const data = {
-        upperBound: state.upperBound,
-        lowerBound: state.lowerBound,
-      };
-      data[state.handle] = value;
-      this.triggerEvents('onChange', [data.lowerBound, data.upperBound]);
-    } else {
-      this.triggerEvents('onChange', value);
-    }
+    this.triggerEvents('onChange', this.getValue());
```

Comment:
Here is a problem, see [`setState`](https://github.com/facebook/react/blob/0b29035484f428cb56e7e1c04a88f66ac020d1d4/src/isomorphic/modern/class/ReactComponent.js#L42)